### ISSUE TITLE
Don't subscribe users when checkbox is not checked

### ISSUE THREAD

**Issue by misulicus** [easydigitaldownloads/edd-mailpoet#15]
Test and even when checkbox is not enabled or enabled but not checked during checkout the users will still be added to the global/product specific list.

**Comment by ashleyfae** [comment]
I'm unable to reproduce this.

Note: I do end up on the `WordPress Users` list, but not on the custom newsletter list I've created and chosen for EDD. Having a `WordPress Users` list of all WP users (even those who haven't opted in) is a MailPoet "feature". https://www.mailpoet.com/wordpress-users-list/

**Comment by mihaijoldis** [comment]
I made a purchase with a new user on the site and it got added here https://prnt.sc/uqo0ot

**Comment by ashleyfae** [comment]
Yeah you're in the `WordPress Users` list. That's a list MailPoet automatically creates with **all user accounts** on it. So it's not EDD adding to that list; it's MailPoet.
